### DISCUSSION TITLE
Как передать suggest_apikey?

### DISCUSSION THREAD

**Discussion by SanchoPanzo** [yandex-maps-unofficial/vue-yandex-maps#135] [Q&A (2.x)] [answered]
Для API Геосаджеста требуется отдельный ключ. Как его передать?

`https://api-maps.yandex.ru/2.1/?lang=ru_RU&apikey=<your key for JS API>&suggest_apikey=<your key for Suggest API>.`

**Comment by daniluk4000** [comment, marked as answer]
Привет! 

В новых Картах вы можете использовать:
- `ymaps3.suggest` (немного урезанный)
- Новое API Геосаджеста (рекомендуется): https://yandex.ru/dev/geosuggest/doc/ru/quickstart

Старым API пользоваться нужно перестать.
